src/components/menu/menu.tsx

```diff
@@ -65,6 +65,16 @@ export default function Menu() {
     closeMenu();
   }, []);
 
+  const playLogo = useCallback(() => {
+    const logo = document.getElementById("logo") as HTMLImageElement | null;
```

Comment:
plutot que document.getElementById il faut utiliser une ref https://react.dev/learn/manipulating-the-dom-with-refs#getting-a-ref-to-the-node / https://github.com/LeRetroProjecteur/website/blob/79daf90c9ee7bbf401747cbce9cc1d4902e393d1/src/components/util/get-html.tsx#L12

Reply:
Tu penses que c'est tjrs nécessaire ?